<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordMimicryClass</class>
 <widget class="QMainWindow" name="RecordMimicryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record Mimicry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Исходное видео</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="videoLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="startCalibrationButton">
          <property name="text">
           <string>Калибровка</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startRecordButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Старт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopRecordButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Стоп</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>53</height>
         </size>
        </property>
        <property name="title">
         <string>Выходной файл</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Путь</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputFileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="outputFileButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Режим</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="webcamButton">
           <property name="text">
            <string>Web - камера</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="writeVideoButton">
           <property name="text">
            <string>Запись видео данных</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="readVideoButton">
           <property name="text">
            <string>Чтение видео данных</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Параметры отображения</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="faceKeyPointBox">
           <property name="text">
            <string>Ключевые точки на лице</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="normalizedKeyPointBox">
           <property name="styleSheet">
            <string notr="true">;</string>
           </property>
           <property name="text">
            <string>Нормализованные ключевые точки</string>
           </property>
           <property name="icon">
            <iconset resource="recordmimicry.qrc">
             <normaloff>:/RecordMimicry/test.png</normaloff>:/RecordMimicry/test.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="calibrationKeyPointBox">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Откалиброванные ключевые точки</string>
           </property>
           <property name="icon">
            <iconset resource="recordmimicry.qrc">
             <normaloff>:/RecordMimicry/Vayu.jpg</normaloff>:/RecordMimicry/Vayu.jpg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>10</width>
             <height>10</height>
            </size>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Нормализованные ключевые точки</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="normalizedPointsLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recordmimicry.qrc"/>
 </resources>
 <connections/>
</ui>
